### PR TITLE
Fanvil - Add Country Tone

### PR DESCRIPTION
**Pull request by fouille**

### CODE REVIEW

**Comment by fouille** [comment]
Tested:
* V6 series
* H series
* i Series

File: plugins/wazo_fanvil/common/common.py [hunk 1]
```diff
@@ -124,6 +124,15 @@ def _do_associate(
...
     _ENCODING = 'UTF-8'
     _LOCALE: dict[str, str] = {}
+    _COUNTRY = {
+        'de_DE': '2',
+        'es_ES': '33',
+        'fr_FR': '25',
+        'fr_CA': '18',
+        'it_IT': '21',
+        'nl_NL': '5',
+        'en_US': '11',
```
[the remark this answers is not in the document]
Comment (sduthil) [reply plugins/wazo_fanvil/common/common.py]: Missing locales added

File: plugins/wazo_fanvil/common/common.py [hunk 2]
```diff
@@ -326,6 +335,7 @@ def _add_locale(self, device, raw_config: dict[str, Any]):
         locale = raw_config.get('locale')
         if not locale:
             return
+        raw_config['XX_country'] = self._COUNTRY[locale]
```
[the remark this answers is not in the document]
Comment (sduthil) [reply plugins/wazo_fanvil/common/common.py]: After discussion, `en_US` seemed the least surprising default.

**Review by bloom1 — APPROVED** [review]
Thank you!

**Pull request merged**